Singer.API/Helpers/Extensions/EntityFrameworkCoreExtensions.cs

```diff
@@ -32,25 +30,19 @@ public static WebApplicationBuilder AddApplicationDbContext(this WebApplicationB
 
     public static async Task<WebApplication> MigrateContexts(this WebApplication app)
     {
-        var initialAdminPassword = app.Configuration.GetSection("Application").Get<ApplicationConfig>().InitialAdminUserPassword;
         using var serviceScope = app.Services.GetService<IServiceScopeFactory>().CreateScope();
 
-        var configrationDbContext = serviceScope.ServiceProvider.GetRequiredService<ConfigurationDbContext>();
-        configrationDbContext.Database.Migrate();
-
-        var persistedGrantDbContext = serviceScope.ServiceProvider.GetRequiredService<PersistedGrantDbContext>();
-        persistedGrantDbContext.Database.Migrate();
-
         var applicationDbContext = serviceScope.ServiceProvider.GetRequiredService<ApplicationDbContext>();
-        applicationDbContext.Database.Migrate();
-
-        await Seed.SeedRolesAsync(serviceScope);
-        await Seed.SeedUsersAsync(serviceScope, applicationDbContext, initialAdminPassword);
-        await Seed.CheckRolesAsync(serviceScope, applicationDbContext);
-        Seed.CreateAPIAndClient(configrationDbContext);
-        Seed.SeedIdentityResources(configrationDbContext);
+        await applicationDbContext.Database.MigrateAsync();
+
+        // TODO Reintroduce
+        //await Seed.SeedRolesAsync(serviceScope);
+        //await Seed.SeedUsersAsync(serviceScope, applicationDbContext, initialAdminPassword);
+        //await Seed.CheckRolesAsync(serviceScope, applicationDbContext);
+        //Seed.CreateAPIAndClient(configrationDbContext);
+        //Seed.SeedIdentityResources(configrationDbContext);
```

Comment:
I think this can be removed. Our application shouldn't have to worry about seeding roles or users anymore. We just need to put the expected roles on the controllers